### DISCUSSION TITLE
Question about pset.show()

### DISCUSSION THREAD

**Discussion by GhelbyMF** [Parcels-code/Parcels#1394] [Q&A]
Hello,

I have been using Parcels for my undergraduate thesis and I have a question regarding `pset.show()`. I am using a HYCOM nc file with domain:
north = 65;                 
south = 10;       
west = 110;       
east = 160;  

I noticed that in the latest Parcels version, `pset.show()` returns only a limited area (Fig 1).  Meanwhile, in the older Parcels version it returns a full domain (Fig 2). Is it normal?
I'm afraid there's something wrong with my HYCOM data

n.b. I used `pset.show(domain={'N':65, 'S':10, 'E':160, 'W':110})` for generation of the Fig 2 in the latest Parcels version

Thanks

![Fig 1](https://github.com/OceanParcels/parcels/assets/101785918/e0600dbb-60d6-403f-ae7f-9078caf2ba9e)
![Fig 2](https://github.com/OceanParcels/parcels/assets/101785918/8ed1cd74-d075-4ffb-a8cd-9329b6d0b640)

**Comment by erikvansebille** [comment]
Dear @GhelbyMF, thanks for your question. However, I _strongly_ advise not to use the built-in plotting tools of Parcels for visualisation. They are really very rudimentary and we don't further develop them (in fact, we're planning to move to the `trajan` package for visualisation in the tutorials, see #1319)

See the [working with Parcels output tutorial](https://docs.oceanparcels.org/en/latest/examples/tutorial_output.html) for tips on how to plot particles.
